tests/test_validate_call.py

```diff
@@ -347,32 +462,6 @@ def foo(args: int, kwargs: int):
     ]
 
 
-def test_v_args():
-    @validate_call
-    def foo1(v__args: int):
-        return v__args
-
-    assert foo1(123) == 123
-
-    @validate_call
-    def foo2(v__kwargs: int):
-        return v__kwargs
-
-    assert foo2(123) == 123
-
-    @validate_call
-    def foo3(v__positional_only: int):
-        return v__positional_only
-
-    assert foo3(123) == 123
-
-    @validate_call
-    def foo4(v__duplicate_kwargs: int):
-        return v__duplicate_kwargs
-
-    assert foo4(123) == 123
-
-
```

Comment:
Why remove this?

Reply:
This looks like an incomplete/unnecessary test to me — the functions are nothing special and basially the same as simply
```py
@validate_call
def foo(x: int):
    return x
```
which obviously have been covered in other tests.

Reply:
It seems these are from `test_deprecated_validate_arguments.py`, where the argument names are invalid at the previous version. But in the new `validate_call` they are just normal names.

Reply:
Yes can be removed after looking at the git blame.